src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -84,8 +84,11 @@ public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.G
         public void Insert(int index, uint item) { }
         [System.CLSCompliantAttribute(false)]
         public void Insert(int index, ulong item) { }
+        public int LastIndexOf(System.Text.Json.JsonNode item) { throw null; }
         public bool Remove(System.Text.Json.JsonNode item) { throw null; }
         public void RemoveAt(int index) { }
+        public void Sort() { }
```

Comment:
For this Sort don't you need the JsonNode to be comparable?

Reply:
Oh yes, you are right.

Reply:
Nah, just that a "default" comparer exist.  It doesn't have to be self-compare, and it doesn't have to be public.

Reply:
```C#
public void Sort() => Sort(JsonNodeComparer.Instance);
```

Reply:
So I should create `JsonNodeComparer` class?